src/index.js

```diff
@@ -12,9 +12,9 @@ searchButton.addEventListener("click", async () => {
   clearInput()
 })
 
-document.addEventListener("keydown", async ({ key }) => {
-  if (key.toLocaleLowerCase() === "enter" && inputBar.value.length > 0) {
-    await fetchCity(inputBar.value)
+inputBar.addEventListener("keydown", (e) => {
```

Comment:
You can use "desctructuring" here to get the key `({key}) => {` 
and then you dont have to use `e.key` you can just use `key`